assets/src/editorIndex.js

```diff
@@ -14,6 +14,7 @@ import { TakeactionboxoutBlock } from './blocks/Takeactionboxout/Takeactionboxou
 import { TimelineBlock } from './blocks/Timeline/TimelineBlock';
 import { addSubAndSuperscript } from './RichTextEnhancements';
 import { setupImageBlockExtension } from './ImageBlockExtension';
+import { SocialshareBlock } from './blocks/Socialshare/SocialshareBlock';
```

Comment:
With the exception of last two items, the rest of the list is alphabetized. Can you please move this below `SocialmediaBlock`?